AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java

```diff
@@ -353,8 +353,16 @@ public void onCreate() {
             UIUtils.showThemedToast(this.getApplicationContext(), getString(R.string.user_is_a_robot), false);
         }
 
-        // make default HTML / JS debugging true for debug build
-        if (BuildConfig.DEBUG) {
+        // make default HTML / JS debugging true for debug build and disable for unit/android tests
+        boolean isDebug = false;
+        try {
+            Class.forName ("org.junit.Test");
+        } catch (ClassNotFoundException e) {
+            isDebug = true;
+        }
+        Timber.d("isDebug: %s", Boolean.toString(isDebug));
```

Comment:
untested, I'd be surprised if it didn't work

```suggestion
        Timber.d("isDebug: %b", isDebug);
```